vector/src/main/java/im/vector/activity/VectorMemberDetailsActivity.java

```diff
@@ -1461,12 +1462,18 @@ private void updateUi() {
     private void updatePresenceInfoUi() {
         // sanity check
         if (null != mPresenceTextView) {
-            mPresenceTextView.setText(VectorUtils.getUserOnlineStatus(this, mSession, mMemberId, new SimpleApiCallback<Void>() {
+            String status = VectorUtils.getUserOnlineStatus(this, mSession, mMemberId, new SimpleApiCallback<Void>() {
                 @Override
                 public void onSuccess(Void info) {
                     mPresenceTextView.setText(VectorUtils.getUserOnlineStatus(VectorMemberDetailsActivity.this, mSession, mMemberId, null));
                 }
-            }));
+            });
+            mPresenceTextView.setText(status);
+            if(status.compareToIgnoreCase("online")==0){
```

Comment:
I assume "online" is elsewhere in the code base, so can it please be centralised into a constant and all references replaced with the constant.

Reply:
other online is in string.xml, which is translatable. However, this response is expected from the server.

Reply:
Can you please add 2 code comments:

1. in VectorMemberDetailsActivity.java and 
2. in string.xml to say that the two values need to stay in sync

Reply:
I have added two comments for the "online" usage.